persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -1261,19 +1275,27 @@ escape (DBName s) =
 data PostgresConf = PostgresConf
     { pgConnStr  :: ConnectionString
       -- ^ The connection string.
+    
+    , pgPoolStripes :: Int
+    -- ^ How many stripes to divide the pool into. See "Data.Pool" for details.
+    , pgPoolIdleTimeout :: Integer
```

Comment:
Ideally this would be `NominalDiffTime`. However, NominalDiffTime lacks a Read instance, and `PostgresConf` derives `Read`. I requested a Read instance here: https://github.com/haskell/time/issues/130